episodes/07-thresholding.md

```diff
@@ -348,13 +348,17 @@ plt.xlim(0, 1.0)
 ![](fig/maize-root-cluster-histogram.png){alt='Grayscale histogram of the maize root image'}
 
 The histogram has a significant peak around 0.2, and a second,
-smaller peak very near 1.0.
+shallower peak near 0.6.
 Thus, this image is a good candidate for thresholding with Otsu's method.
 The mathematical details of how this works are complicated (see
 [the scikit-image documentation](https://scikit-image.org/docs/dev/api/skimage.filters.html#threshold-otsu)
 if you are interested),
 but the outcome is that Otsu's method finds a threshold value between
 the two peaks of a grayscale histogram.
+Note there is a third peak very close to 1.0 which corresponds to the white label and disk on the 
+left hand side of the image. It would be a good idea to remove these white pixels 
+before calculating the Otsu threshold. We will discuss how to do this later in the episode but for 
+now we will continue without removing them.
```

Comment:
```suggestion
:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
The histogram of the maize root image may prompt questions from learners about the interpretation of the peaks and the broader region around 0.6. The focus here is on the separation of background and foreground pixel values. We note that Otsu's method does not work well for the image with the shapes used earlier in this episode, as the foreground pixel values are more distributed. These examples could be augmented by a discussion of unimodal, bimodal, and multimodal histograms. While these points can lead to fruitful discussions, the text in this episode attempts to reduce cognitive load and deliberately simplifies the discussion.
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
```

Reply:
:+1: maybe
```diff
-augmented by
+augmented with
```
and
```diff
-fruitful discussions
+fruitful considerations
```
(to avoid some repetition)